PostFreeSurfer/scripts/CreateMyelinMaps_1res.sh

```diff
@@ -108,7 +125,13 @@ for STRING in ${STRINGII} ; do
   Folder=`echo $STRING | cut -d "@" -f 1`
   Mesh=`echo $STRING | cut -d "@" -f 2`
   ROI=`echo $STRING | cut -d "@" -f 3`
-  for STRINGII in MyelinMap@func SmoothedMyelinMap@func MyelinMap_BC@func SmoothedMyelinMap_BC@func corrThickness@shape ; do
+
+  STRINGIIList="corrThickness@shape"
+  if [ "${T2wPresent}" = "YES" ] ; then
+    STRINGIIList+=" MyelinMap@func SmoothedMyelinMap@func MyelinMap_BC@func SmoothedMyelinMap_BC@func"
+  fi
+
+  for STRINGII in $STRINGIIList ; do
```

Comment:
This hasn't been updated to parallel the treatment in CreateMyelinMaps.sh

Reply:
Ops. Done.